tests/unit/test_adapter.py

```diff
@@ -265,7 +265,9 @@ def test_certificate_authentication(self):
             }
         )
         credentials = connection.credentials
+        trino_auth = credentials.trino_auth()
```

Comment:
No need to create a new variable. Inline the operation directly within the assertion

Reply:
updated 👍